zebra-consensus/src/transaction.rs

```diff
@@ -161,6 +161,16 @@ where
 
             if tx.is_coinbase() {
                 check::coinbase_tx_no_prevout_joinsplit_spend(&tx)?;
+            } else {
+                // check the value pool for non-coinbase transactions
+                if tx
+                    .value_balance(&req.known_utxos())
+                    .unwrap()
```

Comment:
Are you sure that the `value_balance` will never have any errors?

(If we `unwrap` an error here, Zebra will panic.)